test/pylib/manager_client.py

```diff
@@ -374,20 +379,21 @@ async def servers_add(self, servers_num: int = 1,
                           seeds: Optional[List[IPAddress]] = None,
                           driver_connect_opts: dict[str, Any] = {},
                           expected_error: Optional[str] = None,
-                          server_encryption: str = "none") -> List[ServerInfo]:
+                          server_encryption: str = "none",
+                          endpoint_snitch: Optional[str] = None) -> List[ServerInfo]:
         """Add new servers concurrently.
         This function can be called only if the cluster uses consistent topology changes, which support
         concurrent bootstraps. If your test does not fulfill this condition and you want to add multiple
         servers, you should use multiple server_add calls."""
         assert servers_num > 0, f"servers_add: cannot add {servers_num} servers, servers_num must be positive"
 
         try:
-            data = self._create_server_add_data(None, cmdline, config, property_file, start, seeds, expected_error, server_encryption, None)
+            data = self._create_server_add_data(None, cmdline, config, property_file, start, seeds, expected_error, server_encryption, None, endpoint_snitch)
             data['servers_num'] = servers_num
             server_infos = await self.client.put_json("/cluster/addservers", data, response_type="json",
                                                       timeout=ScyllaServer.TOPOLOGY_TIMEOUT * servers_num)
         except Exception as exc:
-            raise Exception("Failed to add servers") from exc
+            raise Exception(f"Failed to add servers: {exc}") from exc
```

Comment:
Doesn't raise-from includes the original `exc` into what's thrown in verbose enough manner?

Reply:
yes, I forgot about it

Reply:
It somehow wasn't very helpful during debugging.
Maybe it was just a pilot error.
Anyhow, I'll remove this hunk and rebase